test/integration/confirmations/signatures/personalSign.test.tsx

```diff
@@ -0,0 +1,259 @@
+              signature_type: ApprovalType.PersonalSign,
+            },
+          }),
+        ]),
+      );
+    });
+
+    fireEvent.click(
+      getByTestId('header-info__account-details-modal__close-button'),
+    );
+
+    await waitFor(() => {
+      expect(
+        queryByTestId('header-info__account-details-modal__account-name'),
+      ).not.toBeInTheDocument();
+    });
+  });
+
+  it('displays the expected title data', async () => {
+    const account =
+      mockMetaMaskState.internalAccounts.accounts[
+        mockMetaMaskState.internalAccounts
+          .selectedAccount as 'cf8dace4-9439-4bd4-b3a8-88c821c8fcb3'
+      ];
+
+    const pendingPersonalSignTime = new Date().getTime();
+    const pendingPersonalSignId = '0050d5b0-c023-11ee-a0cb-3390a510a0ab';
+    const mockedMetaMaskState = {
+      ...mockMetaMaskState,
+      preferences: {
+        ...mockMetaMaskState.preferences,
+        redesignedConfirmationsEnabled: true,
+      },
+      unapprovedPersonalMsgs: {
+        [pendingPersonalSignId]: {
+          id: pendingPersonalSignId,
+          status: 'unapproved',
+          time: pendingPersonalSignTime,
+          type: 'personal_sign',
+          securityProviderResponse: null,
+          msgParams: {
+            from: account.address,
+            data: '0x4578616d706c652060706572736f6e616c5f7369676e60206d657373616765',
+            origin: 'https://metamask.github.io',
+            siwe: { isSIWEMessage: false, parsedMessage: null },
+          },
+        },
+      },
+      unapprovedPersonalMsgCount: 1,
+      pendingApprovals: {
+        [pendingPersonalSignId]: {
+          id: pendingPersonalSignId,
+          origin: 'origin',
+          time: pendingPersonalSignTime,
+          type: ApprovalType.PersonalSign,
+          requestData: {},
+          requestState: null,
+          expectsResult: false,
+        },
+      },
+      pendingApprovalCount: 1,
+    };
+
+    const { getByText } = await integrationTestRender({
+      preloadedState: mockedMetaMaskState,
+      backgroundConnection: backgroundConnectionMocked,
+    });
+
+    expect(getByText('Signature request')).toBeInTheDocument();
+    expect(
+      getByText(
+        'Only confirm this message if you approve the content and trust the requesting site.',
+      ),
+    ).toBeInTheDocument();
+  });
+
+  it('displays the MMI header warning when account signing is not the same as the account selected', async () => {
+    const account =
+      mockMetaMaskState.internalAccounts.accounts[
+        '07c2cfec-36c9-46c4-8115-3836d3ac9047'
+      ];
+    const selectedAccount =
+      mockMetaMaskState.internalAccounts.accounts[
+        mockMetaMaskState.internalAccounts
+          .selectedAccount as 'cf8dace4-9439-4bd4-b3a8-88c821c8fcb3'
+      ];
+
+    const pendingPersonalSignTime = new Date().getTime();
+    const pendingPersonalSignId = '0050d5b0-c023-11ee-a0cb-3390a510a0ab';
```

Comment:
Should we define file level constants for these as I think we use them a few times? The same could be argued for any inline raw strings?